### ISSUE TITLE
A&R -> Variance search improvement

### ISSUE THREAD

**Issue by meyerdarcie** [bcgov/bcparks-ar-admin#355]
#### Description:

Searching variances by individual park and month is time consuming. We need to be able to search variances by bundle (some can contain up to 12 parks) or a range of months

This ticket is to look at the ability to search variances by bundle. 

![Image](https://github.com/user-attachments/assets/8842b22d-c398-43bf-b8f1-e9575e63dfe6)

#### Acceptance Criteria:
[Note: Use 'Given/When/Then' format if it makes sense to. Otherwise, a simple checklist that can be tested.]

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by LindsayMacfarlane** [comment]
Backlog refinement (October 8): This ticket to be implemented for spring season.

**Comment by LindsayMacfarlane** [comment]
May 14th A&R check-in with PROS team. This was flagged as a med-lower priority item to solve. The team indicated that as it is currently set up, it isn't functional. They have work arounds in the interim.
